@navikt/ds-react/src/form/ErrorMessage.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import cl from "classnames";
+import { Label } from "../index";
```

Comment:
```suggestion
import { Label } from "..";
```

Denne endringen kan gjøres mange steder.

Reply:
Fikser i en egen PR